blocks/market-commentary/market-commentary.js

```diff
@@ -98,7 +101,8 @@ function updateDots(block) {
 
 async function generateCardsView(block, placeholders) {
   const blogsContainer = block.querySelector('.market-commentary-track');
-  fetchData(`${getHostUrl()}/scripts/mock-commentarydata.json`, async (error, marketCommentaryDataArray = []) => {
+  getDataFromAPI(getResearchAPIUrl(), 'GetResearchEquityMarketCommentary', async (error, marketCommentaryDataArray = []) => {
```

Comment:
It would be good if we don't need to fetch the url here.
API names are fixed on the basis of that we can identify the url in method ` getDataFromAPI`

Reply:
if more APIs come in marketing namespace or any other we will too many if and else in getDataFromAPI.
Keeping everything simple

Reply:
we can switch case but anyway this is also good. less code in case of new api